components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -168,6 +168,13 @@ export default function ProjectSettingsView() {
                 />
                 {enablePrebuilds && (
                     <>
+                        <CheckboxInputField
+                            label="Prebuild the default branch only"
+                            hint="Commits on other branches will be ignored."
+                            disabled={!enablePrebuilds}
+                            checked={project.settings?.prebuildDefaultBranchOnly ?? false}
+                            onChange={(checked) => updateProjectSettings({ prebuildDefaultBranchOnly: checked })}
+                        />
```

Comment:
**thought**: Instead of a checkbox, and in light of the potential upcoming changes in EXP-544, we could use a select element for more flexibility. What do you think? For example, default could be _Default branch only_, other options could include _All branches_, etc.

<img width="1388" alt="prebuilds-strategy" src="https://github.com/gitpod-io/gitpod/assets/120486/d51720be-7163-44f0-9a50-2d419623c63e">

Reply:
done